src/components/GenericError.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Button,
+  ButtonBase,
+  Link,
+  GU,
+  IconDown,
+  RADIUS,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+
+const SUPPORT_URL = 'https://github.com/aragon/court-dashboard/issues/new'
+
+const GenericError = React.memo(function GenericError({
+  detailsTitle,
+  detailsContent,
+  reportCallback,
+}) {
+  const theme = useTheme()
+  const [opened, setOpened] = useState(false)
+  const toggle = useCallback(() => {
+    setOpened(!opened)
```

Comment:
These can usually be streamlined to using the functional setter, which then allows you to never re-create this function:

```
const toggle = useCallback(() => {
  setOpened(opened => !opened)
}, [])
```